src/govuk/components/character-count/character-count.test.js

```diff
@@ -143,6 +145,36 @@ describe('Character count', () => {
       })
     })
 
+    describe('when it has the spellcheck attribute', () => {
```

Comment:
Sorry, these tests should be in `src/govuk/components/character-counte/template.js` – components that have javascript have two test files:

- [component-name].test.js – integration tests that Puppeteer to test that the JavaScript behaves as expected
- template.test.js – unit tests (sort of) that test the HTML outputted by the template given different options

Reply:
🤦‍♀️ Not sure how I didn't spot that, thanks!

Reply:
It's not at all obvious! Is there anywhere you looked where you think it'd make sense to have documented this?